homeassistant/components/growatt_server/config_flow.py

```diff
@@ -5,7 +5,7 @@
 from homeassistant import config_entries
 from homeassistant.const import CONF_NAME, CONF_PASSWORD, CONF_USERNAME
 
-from .const import CONF_PLANT_ID, DOMAIN  # noqa # pylint: disable=unused-import
+from .const import CONF_PLANT_ID, DOMAIN  # noqa: E501 # pylint: disable=unused-import
```

Comment:
We don't need to disable pylint anymore. It has been fixed upstream. Please rebase on latest dev branch if needed.
```suggestion
from .const import CONF_PLANT_ID, DOMAIN
```